src/bin/scheduler/scheduling/scheduler.rs

```diff
@@ -80,7 +93,13 @@ async fn run_cleanup_job(cancellation_token: CancellationToken, plans: Vec<Plan>
             _ = clock.tick() => { }
             _ = cancellation_token.cancelled() => { return }
         };
-        spawn_blocking(move || cleanup_working_directories(plans.iter()));
+        let _ = spawn_blocking(move || cleanup_working_directories(plans.iter()))
+            .await
+            .map_err(|err| {
+                log_and_return_error(
```

Comment:
Would it not be simpler to log the error without returning it?

Reply:
If we decide to change the logging format in `log_and_return_error`, this would be taken into account here.